frontend/src/components/parentData/ParentSeries.js

```diff
@@ -11,7 +11,7 @@ const ParentSeries = () => {
 
     const [
         {
-            parentData: { seriesData, buildData },
```

Comment:
Does setSeriesData work correctly if seriesData is not fetched? Have had issues before with dispatch not working correctly in such situations. Fetching parentData might be enough though.